### DISCUSSION TITLE
Listen to "Save" Image Editor

### DISCUSSION THREAD

**Discussion by alexmf23** [pimcore/pimcore#13801] [Q&A / Support] [answered]
Hello, is there a way to listen to the Save button from the image editor ?

<img width="996" alt="Screenshot 2022-12-14 at 16 50 39" src="https://user-images.githubusercontent.com/63473281/207645065-0769603f-582b-407f-8f9e-47ce19944c78.png">

**Comment by dvesh3** [comment, marked as answer]
Unfortunately, there's no event triggered for this action. you can provide a PR to add `pimcore.admin.asset.imageEditor.preSave` [here ](https://github.com/pimcore/pimcore/blob/2f55f5647b2106ce5649b91f2c2c3b09533da16e/bundles/AdminBundle/src/Controller/Admin/Asset/AssetController.php#L1808)
